### ISSUE TITLE
Delete `hasBiologicalOffspring`

### ISSUE THREAD

**Issue by rjyounes** [semanticarts/gist#638]
And change the restriction on LivingThing:

```
			[
				a owl:Restriction ;
				owl:onProperty gist:hasBiologicalParent ;
				owl:someValuesFrom gist:LivingThing ;
			]
```
to

```
			[
				a owl:Restriction ;
				owl:onProperty gist:hasBiologicalParent ;
				owl:minCardinality "1"^^xsd:nonNegativeInteger ;
			]
```
`gist:rangeIncludes gist:Person` also seems unnecessary in that case.

**Comment by uscholdm** [comment]
What non-living thing(s) do you have in mind that can be a biological parent?
If you really think that is possible, then its best to use someValuesFrom owl:Thing rather than a min 1, which reduces flexibility and can cause inference problems in some cases, the ones you least expect to getcha  :-)

**Comment by rjyounes** [comment]
What in my comments suggests that non-living things can have a biological parent? I certainly do not think that.

**Comment by rjyounes** [comment]
Summary of changes in gist 11
=====

Note: doesn't exclude changes that require no migration, such as modifying annotations and adding or removing a domain or range (even though these affect inferencing).

New time model
-----

Most of the model changes probably should be done manually, because it's not a simple renaming. Object properties have to be renamed to datatype properties, but the properties of the TimeInstant object determine which datatype property to use. It could be automated but would take some work.

### Deleted

(I think these get added to `detect_removed.rq`.)

#### Classes

```markdown

TimeInstant
DateInstant
GreenwichInstant
HumanInstant
LocalInstant
SystemInstant

```

#### Properties

```markdown

hasEnd
hasActual
hasActualEnd
hasPlannedEnd
hasDeathDate
hasStart
hasActualStart
hasPlanned
hasBirthDate
hasPlannedStart
epoch
localDate
localDateTime
localTime
universalDate
universalDateTime
universalTime
isSameTimeAs
wasLastModifiedAt

```

### Modified

- `ContemporaneousEvent` renamed to `ContemporaryEvent`
- `usesTimeZoneStandard` - domain `TimeInstant` removed

### Added

```markdown

atDateTime
birthDate
deathDate
endDateTime
actualEndDateTime
actualEndDate
actualEndMicrosecond
actualEndMinute
actualEndYear
plannedEndDateTime
plannedEndDate
plannedEndMinute
plannedEndYear
startDateTime
actualStartDateTime
actualStartDate
actualStartMicrosecond
actualStartMinute
actualStartYear
plannedStartDateTime
plannedStartDate
plannedStartMinute
plannedStartYear

```

#### Notes on datetime migration

Tricky but doable with fairly large effort. Example: from triples

```markdown

:_x gist:hasPlannedEnd :_ti .
:_ti a gist:DateInstant ;
   gist:localDateTime "2022-03-16T01:01:01Z-04:00"^^xsd:datetime ;
   .

```

You create:

```markdown

:_x gist:plannedEndDate "2022-03-16T00:00:00Z-04:00"^^xsd:datetime .

```

Note that hours, minutes, and seconds must be zeroed out for date values. There are a lot of wrinkles. For example, the local datetime may not have an offset, but a value for `gist:usesTimeZoneStandard` instead. I can give you more details if you want to write this script, but we might be better off to just provide some sample triples like the ones above instead, trying to cover a number of different cases.

Miscellaneous
-----

### Deleted

- `hasOrderedMember`
- `hasBiologicalOffspring`

### Modified

- `hasJurisdictionOver` renamed TBD

**Comment by rjyounes** [comment]
Fixed in 11.0.0.
